components/partials/portfolio/bridge/token-selector/select.vue

```diff
@@ -51,11 +51,12 @@
                 :valid="valid"
                 :max="balanceToFixed"
                 :max-decimals="value.decimals"
-                :max-selector="balance.gt(0.01)"
+                :max-selector="maxSelector && balance.gt(0.01)"
                 :value="amount"
                 @input="handleAmountChange"
                 @blur="resetIsSearching"
                 @mousedown.native.stop="focusInput"
+                @input-max="() => onMaxInput(100)"
```

Comment:
since max will always be 100, let's not pass it as a parameter.
`@input-max="onMaxInput"`

more comments `~/components/partials/swap/index.vue` file